ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -566,12 +576,12 @@ aaa_accounting:
   exec:
     default:
       type: < none | start-stop | stop-only >
-      group: < group_name >
+      group: < group name >
```

Comment:
Change back to `group_name`

Reply:
exec:
    default:
      type: < none | start-stop | stop-only >
      group: < group name >

group is invalid, removing

Reply:
Good point, fixed!